packages/office-ui-fabric-react/src/components/ScreenReaderAlert/ScreenReaderAlertPage.tsx

```diff
@@ -0,0 +1,121 @@
+              code={ ScreenReaderAlertRepeatExampleCode as string }
+            >
+              <ScreenReaderAlertRepeatExample />
+            </ExampleCard>
+            <ExampleCard
+              title='ScreenReaderAlert with Spinner'
+              code={ ScreenReaderAlertSpinnerExampleCode as string }
+            >
+              <ScreenReaderAlertSpinnerExample />
+            </ExampleCard>
+            <ExampleCard
+              title='ScreenReaderAlert polite mode'
+              code={ ScreenReaderAlertAfterExampleCode as string }
+            >
+              <ScreenReaderAlertAfterExample />
+            </ExampleCard>
+          </div>
+        }
+        propertiesTables={
+          <PropertiesTableSet
+            sources={ [
+              require<string>('!raw-loader!office-ui-fabric-react/src/components/ScreenReaderAlert/ScreenReaderAlert.Props.ts')
+            ] }
+          />
+        }
+        overview={
+          <div>
+            <p>
+              ScreenReaderAlerts are used to read out the notifications on the page to the blind people with the help of screen readers like JAWS, Chrome VOX and NVDA. It is implemented by live region and overcomes some known issues about live region when using with React.
+            </p>
+            <p>
+              It renders an addition element for screen reader only, so using this component while the text/ui is visible will duplicate the text in the DOM. You should use vanilla live region attributes instead if you don't want this duplicate element.
+            </p>
+            <p>
+              This component has been tested through following combination of screen readers and browsers:
+            </p>
+            <ul>
+              <li>
+                1. <strong>Narrator in windows 10 Enterprise 14393.577</strong> with <strong>Edge 38.14393.0.0</strong>.
+              </li>
+              <li>
+                2. <strong>NVDA 2016.4</strong> with <strong>FireFox v50.1.0</strong>.
+              </li>
+              <li>
+                3. <strong>Chrome VOX v53</strong> with <strong>Chrome v54</strong>.
+                <br />
+                Note: ReadingMode.ReadAfterOtherContent option is currently not supported in ChromeVOX, text will be read immediately.
+              </li>
+              <li>
+                4. <strong>JAWS 18</strong> with <strong>IE 11</strong>.
+              </li>
+            </ul>
+
+            <p>
+              To try the demo in this page, you should at least using one kind of screen reader. E.g. JAWS, ChromeVOX, NVDA.
+              Better use the browser recommended by the screen reader you're using. E.g. IE11 with JAWS, Chrome with ChromeVOX, FireFox with NVDA.
+            </p>
+          </div>
+        }
+        bestPractices={
+          <div></div>
+        }
+        dos={
+          <div>
+            <ul>
+              <li>Use if it would be hard for blind people to know dynamic changes on the page.</li>
```

Comment:
@dzearing 

> In your PR you here you also have an auto clear text behavior, which the spinner PR doesn't. Is that a bug in the spinner PR? I don't understand what the purpose is of the auto clearing.
The auto clearing comes from @cschlechty, that the alert text should not be persisted in the DOM. Blind people have capibility of navigating the DOM elements, this kind of elements will be confusing to blind people. As the low possibility been navigated by blind people, IMO, I don't think it's a bug if spinner doesn't implement it.